debian/changelog

```diff
@@ -1,3 +1,10 @@
+cloud-init (24.1.3-0ubuntu4) noble; urgency=medium
+
+  * cherry-pick a6f7577d: fix(package_update): avoid snap refresh in
```

Comment:
`bug` is what committed, so we should keep this as the same title as in the original commit.